### PR TITLE
Use --- instead of —

### PR DESCRIPTION
**Pull request by KnorpelSenf**

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
It's currently inconsistent

**Comment by roj1512** [comment]
What do you mean? Isn’t this PQ ready?

**Comment by KnorpelSenf** [comment]
Yes, it is ready. I was adding additional information why this change was necessary.

**Comment by roj1512** [comment]
Is it translatable?

**Comment by KnorpelSenf** [comment]
Yes. I do not know the rules for em dashes in other languages.

For example, — is never used in German, only – is found.

**Review by roj1512 — APPROVED** [review]

**Comment by roj1512** [comment]
@habemuscode Any changes to be made in Spanish?

**Comment by roj1512** [comment]
@Nazar-Ant Any changes to be made in Ukrainian?

**Comment by Nazar-Ant** [comment]
@roj1512 Yes, we decided to also use a triple hyphen to represent dashes. This is something @LWJerri is working on.

File: site/docs/uk/guide/introduction.md [hunk 1]
```diff
@@ -45,8 +45,8 @@ grammY — це бібліотека, яка спрощує створення 
 
 ```asciiart:no-line-numbers
 _________          надсилає           ____________                  ___________
-| Аліса | —> повідомлення Telegram —> | Telegram | —> HTTP запит —> | ваш бот |
-—————————        вашому боту          ————————————                  ———————————
+| Аліса | ---> повідомлення Telegram ---> | Telegram | ---> HTTP запит ---> | ваш бот |
+---------------------------        вашому боту          ------------------------------------                  ---------------------------------
```
Comment (LWJerri) [inline site/docs/uk/guide/introduction.md:49]: @Nazar-Ant, @roj1512, can I do this?

File: site/docs/uk/guide/middleware.md [hunk 2]
```diff
@@ -76,11 +76,11 @@ type NextFunction = () => Promise<void>;
...
 ```asciiart:no-line-numbers
 (ctx, next) => ...    |
-(ctx, next) => ...    |—————вищий проміжний обробник відносно X
+(ctx, next) => ...    |---------------вищий проміжний обробник відносно X
 (ctx, next) => ...    |
-(ctx, next) => ...       <— проміжний обробник X. Викликаємо `next`, щоб передати оновлення 🡳
+(ctx, next) => ...       <--- проміжний обробник X. Викликаємо `next`, щоб передати оновлення 🡳
 (ctx, next) => ...    |
-(ctx, next) => ...    |—————нижчий проміжний обробник відносно X
+(ctx, next) => ...    |---------------нижчий проміжний обробник відносно X
```
Comment (LWJerri) [inline site/docs/uk/guide/middleware.md:83]: And here?

**Review by LWJerri — APPROVED** [review]

**Comment by LWJerri** [comment]
> 🚀 Deployed on https://64920538f6001c1dc38e48fa--grammy.netlify.app

@KnorpelSenf, why we don't have a fresh build?

**Comment by CikiMomogi** [comment]
> > rocket Deployed on https://64920538f6001c1dc38e48fa--grammy.netlify.app
> 
> @KnorpelSenf, why we don't have a fresh build?

The conflicts must be resolved first.

**Comment by CikiMomogi** [comment]
Oops the branch was changed. Sorry my bad.

**Comment by LWJerri** [comment]
@CikiMomogi, I think changes for long dash doesn't apply to code block's.

**Comment by LWJerri** [comment]
> @CikiMomogi, I think changes for long dash doesn't apply to code block's.

1. https://6497d2cb7a500b538129bdf4--grammy.netlify.app/uk/guide/middleware.html#%D1%81%D1%82%D0%B5%D0%BA-%D0%BF%D1%80%D0%BE%D0%BC%D1%96%D0%B6%D0%BD%D0%B8%D1%85-%D0%BE%D0%B1%D1%80%D0%BE%D0%B1%D0%BD%D0%B8%D0%BA%D1%96%D0%B2
2. https://6497d2cb7a500b538129bdf4--grammy.netlify.app/uk/guide/introduction.html#%D1%8F%D0%BA-%D0%BF%D1%96%D0%B4%D1%82%D1%80%D0%B8%D0%BC%D1%83%D0%B2%D0%B0%D1%82%D0%B8-%D1%80%D0%BE%D0%B1%D0%BE%D1%82%D1%83-%D0%B1%D0%BE%D1%82%D0%B0

Scroll down and find tables.

**Comment by LWJerri** [comment]
OMG, I missed open updated links, xDDD.....

**Comment by LWJerri** [comment]
image

Incorrect long dash render. (https://github.com/grammyjs/website/blob/fix-conflict-triple-dash/site/docs/uk/guide/middleware.md, 77-85 lines).

**Comment by CikiMomogi** [comment]
> @CikiMomogi, I think changes for long dash doesn't apply to code block's.

Indeed. All characters written inside the code block will be rendered as they are.

**Comment by LWJerri** [comment]
> > @CikiMomogi, I think changes for long dash doesn't apply to code block's.
> 
> Indeed. All characters written inside the code block will be rendered as they are.

Oh, okay, thanks.

**Review by Nazar-Ant — CHANGES_REQUESTED** [review]

File: site/docs/uk/guide/introduction.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ grammY — це бібліотека, яка спрощує створення 
 ```asciiart:no-line-numbers
 _________          надсилає           ____________                  ___________
 | Аліса | —> повідомлення Telegram —> | Telegram | —> HTTP запит —> | ваш бот |
-—————————        вашому боту          ————————————                  ———————————
+_________        вашому боту          ____________                  ___________
```
Comment (Nazar-Ant) [inline site/docs/uk/guide/introduction.md:49]: Що тут не так було?

**Review by habemuscode — APPROVED** [review]

File: site/docs/uk/guide/introduction.md (hunk 3 above)
Comment (LWJerri) [reply site/docs/uk/guide/introduction.md:49]: > Що тут не так було?

https://6497d2cb7a500b538129bdf4--grammy.netlify.app/uk/guide/introduction.html#%D1%8F%D0%BA-%D0%BF%D1%96%D0%B4%D1%82%D1%80%D0%B8%D0%BC%D1%83%D0%B2%D0%B0%D1%82%D0%B8-%D1%80%D0%BE%D0%B1%D0%BE%D1%82%D1%83-%D0%B1%D0%BE%D1%82%D0%B0

File: site/docs/uk/guide/middleware.md [hunk 4]
```diff
@@ -76,11 +76,11 @@ type NextFunction = () => Promise<void>;
 
 ```asciiart:no-line-numbers
 (ctx, next) => ...    |
-(ctx, next) => ...    |—————вищий проміжний обробник відносно X
+(ctx, next) => ...    |———вищий проміжний обробник відносно X
 (ctx, next) => ...    |
 (ctx, next) => ...       <— проміжний обробник X. Викликаємо `next`, щоб передати оновлення 🡳
 (ctx, next) => ...    |
-(ctx, next) => ...    |—————нижчий проміжний обробник відносно X
+(ctx, next) => ...    |———нижчий проміжний обробник відносно X
```
Comment (LWJerri) [inline site/docs/uk/guide/middleware.md:83]: Fix it.

File: site/docs/uk/guide/deployment-types.md [hunk 5]
```diff
@@ -42,14 +42,14 @@ _Наступного дня вам знову хочеться того сма
 ```asciiart:no-line-numbers
 ______________                                  _____________
 |            |                                  |           |
-|            |   <--- чи є повідомлення? ---    |           |
-|            |    ---        ні.         --->   |           |
+|            |    <—  чи є повідомлення?  —     |           |
+|            |     —         ні.          —>    |           |
```
Comment (LWJerri) [inline site/docs/uk/guide/deployment-types.md:46]: Fix it.

**Review by roj1512 — APPROVED** [review]

**Pull request merged**